src/vs/workbench/api/browser/mainThreadAuthentication.ts

```diff
@@ -112,6 +112,12 @@ export class MainThreadAuthenticationProvider extends Disposable implements IAut
 			await Promise.all(removeSessionPromises);
 			removeAccountUsage(this.storageService, this.id, accountName);
 			this.storageService.remove(`${this.id}-${accountName}`, StorageScope.APPLICATION);
+
+			// {{SQL CARBON EDIT}} - BEGIN - Invoking updateAccountList delegate if a callback definition is defined.
+			if (updateAccontList) {
+				await updateAccontList();
```

Comment:
Why do we need this callback? Couldn't the caller just await on this function and then call whatever they want to after (such as refreshing the account list)?